share/models/logs.py

```diff
@@ -107,6 +108,10 @@ class Meta:
         abstract = True
         ordering = ('-date_modified', )
 
+    @property
+    def status_identifier(self):
+        return next(k for k, v in type(self).STATUS._identifier_map.items() if v == self.status)
```

Comment:
I'm mostly just using it for the color map in the admin. Seemed a bit nicer, should that just be changed over as well?